frontend/src/Components/CreateItemForm.js

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import {Button,Container,Grid,TextField} from '@mui/material';
+
+
+export default function CreateItemForm(props){
+    const [newToDoName,setNewToDoName] = useState('');
+    const handleChange = (event) =>{
+        setNewToDoName(event.target.value);
+    }
+
+    const handleKeyPress = (key, newTodoName)=>{
+        if(key === 'Enter'){
+            handleSubmit(newTodoName);
+        }
+    }
+
+    const handleSubmit = (newToDoName)=>{
+        const trimmedToDo = newToDoName !== null && newToDoName !== undefined ? newToDoName.trim() : null;
```

Comment:
Since JS uses truthy/falsey logic, this could be simplified to:

```suggestion
        const trimmedToDo = newToDoName && newToDoName.trim()
```

but either way all of this could be avoided by adding an early return to the start of the function like:
```js
if (!newToDoName)
  return
```